tests/system/test_limit_eps/test_limit_eps.py

```diff
@@ -0,0 +1,193 @@
+import os
```

Comment:
Migrate to the new QA integration framework used in the rest of the API tests. Here's an [example](https://github.com/wazuh/wazuh/blob/master/tests/integration/test_api/test_config/test_upload_configuration/test_upload_configuration.py).